src/apis/us/UserCommHandler.java

```diff
@@ -0,0 +1,43 @@
+package apis.us;
+
+public interface UserCommHandler {
+
+    String outPutLocation=null;
```

Comment:
Need to double check how Java handles file paths. There might be a built in way to handle it instead of Strings; look into the Java Path interface and things that use it when you get a chance: https://docs.oracle.com/javase/8/docs/api/java/nio/file/Path.html

Reply:
Okay! I feel like strings might be easier to pass around, here is an exceprt from the documunatation "WARNING: This interface is only intended to be implemented by those developing custom file system implementations. Methods may be added to this interface in future releases."

We can still switch, but I think path objects would be an extra later of complicated. Especially casting from path to int to string to path, etc.